localization/en/strings.json

```diff
@@ -1,4 +1,8 @@
 {
+    "file_size": "File size of __size__",
+    "finished": "Finished",
+    "calculating": "Calculating...",
+    "calc_size": "Calculate Total Size",
```

Comment:
Maybe group these under "workshop"? Also "File size of" sounds like the file size of a file, instead of the size, so something like "File size:" should do.

Reply:
The reason why I didn't do `:` was because it didn't fit consistently with the other details shown here: 
![image](https://user-images.githubusercontent.com/4411977/74087695-174e2f00-4a8f-11ea-9152-ef4749461baf.png)

If you know another, better way of phrasing it, let me know.

Reply:
Just "File size" then?

Reply:
Hmm, perhaps

Reply:
Actually, I think "File size is xxx" would work best. As for your grouping idea, I wouldn't do this. These labels are still general enough to be used for other applications.